functions/apply-for-certified-contributor.ts

```diff
@@ -111,17 +117,84 @@ async function handler(event) {
+    2. Provenance will lead you through your identity verification process, and ask you to sign an agreement binding you to a code of conduct and non-disclosure. Code4 Corporation and the Code4rena DAO do NOT have access to your personal information, simply the verified knowledge that you have (or have not) been certified.
+    3. Code4 Corporation will contact you to let you know when your application has been approved.
+
+    This process is entirely managed by humans, so it can take a while to complete. We appreciate your patience with the process!
+    
+    The Code4 team
+    `;
+
+  const html =
+    "<p>Thank you for applying to be a certified Code4rena warden! Here's what to expect next:</p>" +
+    "<ol>" +
+    "<li>If you meet the " +
+    '<a href="https://docs.code4rena.com/roles/wardens/certified-wardens#who-is-eligible-to-be-a-certified-contributor">' +
+    "eligibility requirements</a>" +
+    ', the DAO\'s AML/KYC agent, <a href="https://provenance.company/">Provenance</a>, ' +
+    "will contact you to certify your identity. <strong>Please watch for an email that will come directly " +
+    "from Provenance (https://provenance.company/).</strong></li>" +
+    "<li>Provenance will lead you through your identity verification process, and ask you to sign an " +
+    "agreement binding you to a code of conduct and non-disclosure. Code4 Corporation and the Code4rena " +
+    "DAO do NOT have access to your personal information, simply the verified knowledge that you have " +
+    "(or have not) been certified.</li>" +
+    "<li>Code4 Corporation will contact you to let you know when your application has been approved.</li>" +
+    "</ol>" +
+    "<p>This process is entirely managed by humans, so it can take a while to complete. We appreciate your patience with the process!</p>" +
+    "<p>The Code4 team</p>";
+
+  const mailOptions = {
+    from: "submissions@code423n4.com",
+    to: recipients,
+    subject: `Certified warden application for ${ticket.wardenHandle} has been received!`,
+    text,
+    html,
+  };
+
+  const mail = new MailComposer(mailOptions);
+
+  const emailPromise = new Promise((resolve, reject) => {
+    mail.compile().build((err, message) => {
+      if (err) {
+        reject({
+          statusCode: err.status || 500,
+          body: JSON.stringify({ error: err.message }),
+        });
+      }
+
+      const dataToSend = {
+        to: recipients,
+        message: message.toString("ascii"),
+      };
+      mg.messages().sendMime(dataToSend, (sendError, body) => {
+        if (sendError) {
+          reject({
+            statusCode: sendError.status || 500,
+            body: JSON.stringify({ error: sendError.message }),
+          });
+        }
+        resolve({
+          statusCode: 201,
+          body: "Your request has been submitted",
+        });
+      });
+    });
+  });
+
+  return await emailPromise;
```

Comment:
Shouldn't return this?

Reply:
Yes, I did mean to return that. The netlify lambda functions have to return an object with statusCode and body. The promise resolves with the object, so I return the resolved promise. But correct me if there's something I'm misunderstanding here.

Reply:
I see. Maybe it'd make more sense to explicitly return a success object, at the end of everything.

Reply:
Ah, yeah that makes sense.